packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -446,12 +447,13 @@ export class ContextualMenuBase extends React.Component<IContextualMenuProps, IC
 
   private _tryFocusPreviousActiveElement = (options: {
     containsFocus: boolean;
+    documentContainsFocus: boolean;
     originalElement: HTMLElement | Window | undefined;
   }) => {
     if (this.props.onRestoreFocus) {
       this.props.onRestoreFocus(options);
     } else {
-      if (options && options.containsFocus && this._previousActiveElement) {
+      if (options && options.containsFocus && options.documentContainsFocus && this._previousActiveElement) {
```

Comment:
Can we remove this check here? Because you can pass in your own this check isn't needed.